scripts/install_dev_server.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
AFAICS all you need to do is move your linking into install_from_src and put it under a LINK_TO_SRC variable that is false by default. There's too much duplication here to warrant a whole new script. There's already a fairly bewildering array of them.